protos/robots/nubots/NUgus/NUgus_mesh/NUgusMain.proto

```diff
@@ -0,0 +1,1716 @@
+            ]
+          }
+        }
+      }
+      HingeJointWithBacklash {
+        backlash IS backlash
+        gearMass IS gearMass
+        jointParameters HingeJointParameters {
+          axis 0.000000 0.000000 1.000000
+          anchor 0.000000 0.000000 0.260000
+          dampingConstant IS MX64-damping
+          staticFriction IS MX64-friction
+        }
+        device [
+          RotationalMotor {
+            name "neck_yaw"
+            maxVelocity IS MX64-vel
+            minPosition -3.14159265359
+            maxPosition 3.14159265359
+            maxTorque IS MX64-torque
+          }
+          PositionSensor {
+            name "neck_yaw_sensor"
+            resolution IS DYNAMIXEL-RESOLUTION
+          }
+        ]
+        endPoint Solid {
+          translation 0.000000 0.000000 0.260000
+          rotation 0.000000 0.000000 1.000000 1.570790
+          children [
+            DEF neck NUgus_neckMesh {
+              robotTexture IS robotTexture
+            }
+            HingeJointWithBacklash {
+              gearMass IS gearMass
+              backlash IS backlash
+              jointParameters HingeJointParameters {
+                axis 1.000000 0.000000 0.000000
+                anchor 0.000000 0.000000 0.000000
+                dampingConstant IS MX64-damping
+                staticFriction IS MX64-friction
+              }
+              device [
+                RotationalMotor {
+                  name "head_pitch"
+                  maxVelocity IS MX64-vel
+                  minPosition -3.14159265359
+                  maxPosition 3.14159265359
+                  maxTorque IS MX64-torque
+                }
+                PositionSensor {
+                  name "head_pitch_sensor"
+                  resolution IS DYNAMIXEL-RESOLUTION
+                }
+              ]
+              endPoint Solid {
+                translation 0.000000 0.000000 0.000000
+                rotation 0.000000 1.000000 0.000000 0.000000
+                children [
+                  DEF head NUgus_headMesh {
+                    robotTexture IS robotTexture
+                  }
+                  Transform {
+                    translation 0.000000 0.000000 0.000000
+                    rotation 0.000000 0.000000 1.000000 -1.570790
+                    children [
+                      DEF left_camera Camera {
+                        name "left_camera"
+                        translation 0.0889361 0.0352991 0.0713675
+                        rotation 0.0 0.0 1.0 0.0
+
+                        # Set parameters from fields
+                        fieldOfView IS fieldOfView
+                        width IS cameraWidth
+                        height IS cameraHeight
+                        noise IS cameraNoise
+                        motionBlur IS cameraMotionBlur
+                        spherical FALSE
+
+                        # Allows the robot to detect objects in the world
```

Comment:
please add a parameter to procedually (i.e. with lua or javascript in the proto) remove the recognition.